src/include/duckdb/common/vector_operations/aggregate_executor.hpp

```diff
@@ -382,15 +382,74 @@ class AggregateExecutor {
 		}
 	}
 
+	using Frames = vector<FrameBounds>;
```

Comment:
Nit: can we use `vector<FrameBounds>`?

Reply:
As I mentioned, I find `using` to be good documentation.

Reply:
But if @Mytherin disagrees I can certainly change it.